components/ecu/ecu_firmware/Core/Src/can.c

```diff
@@ -4,21 +4,38 @@
  * 
  * @date 2021/01/30
  * @author Blake Shular (blake-shular)
- */
+
+*/
 
 #include "can.h"
+#include "main.h"
+
+#define CAN_ID 0x450
+#define MESSAGE_LENGTH 3
+
 
 CAN_TxHeaderTypeDef current_message_header = {
-    0x624, 0, CAN_ID_STD, CAN_RTR_DATA, 6, DISABLE
+    CAN_ID, 0, CAN_ID_STD, CAN_RTR_DATA, MESSAGE_LENGTH, DISABLE
 }; 
 
-void CAN_send_current(int currentVal) {
+/*
```

Comment:
Doxygen comments need 2 asterisks on the first line, see below. You should notice when you do this that VS Code will highlight certain parts of the comment differently.

Also, please add a one line description of what the function does at the top of this comment (also see below).

```suggestion
/**
 * @brief Transmit ECU current reading CAN message
 *
```